addon/ng2/models/webpack-build-common.ts

```diff
@@ -0,0 +1,92 @@
+import * as webpack from 'webpack';
+import {LoaderConfig} from '../utilities/ts-path-mappings-webpack-plugin';
+
+const path = require('path');
+const ForkCheckerPlugin = require('awesome-typescript-loader').ForkCheckerPlugin;
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+export const getWebpackCommonConfig = function(projectRoot: string) {
```

Comment:
webpack 2.x typings have not been created otherwise I would, however there are all sorts of small changes that will appear as typings errors. Thus why I am using `any` in webpack-config.ts This is being woked on though.